<?xml version="1.0"?>
<launch>
    <node pkg="ndt_mcl" type="3d_ndt_mcl_node" required="true" name="NDTMCL3D">
	<!--<param name="points_topic" value="/velodyne_points" />-->
	<param name="points_topic" value="/velotime_points" />
	<!--<param name="odometry_topic" value="/RosAria/pose" />-->
	<param name="odometry_topic" value="odometry" />
	<!--<param name="map_resolution" value="0.4" />-->
	<param name="map_resolution" value="1.6" />
	<!-- Specific sensor offset parameters for the example file -->
	<param name="set_sensor_pose" value="true" />
	<param name="sensor_pose_x" value="0.0" />
	<!--<param name="sensor_pose_z" value="-3.8" />-->
	<param name="sensor_pose_z" value="-5.5" />
	<param name="sensor_pose_t" value="0.0" />

	<param name="set_initial_pose" value="true" />
	<param name="pose_init_x" value="-0.80" />
	<param name="pose_init_y" value="0.0" />
	<param name="pose_init_t" value="0.0" />
	
	<param name="forceSIR" value="false" />
	<param name="load_map_from_file" value="true" />
	<!--<param name="map_file_name" value="$(find ndt_fuser)/maps/skygarden1_map.jff" />-->
	<param name="map_file_name" value="$(find ndt_fuser)/maps/dover_map.jff" />
	<param name="save_output_map" value="false" />
	
     </node>   
</launch>
